recipes/virtual_dataframe/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "virtual_dataframe" %}
+{% set version = "v0.5rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pprados/virtual_dataframe/tarball/{{ version }}/virtual_dataframe.tar.gz
+  sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+build:
+  noarch: python  # Impossible with entry_points. See https://github.com/conda/conda-build/issues/2627
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install --progress-bar off --quiet --no-input . -vv
+  entry_points:
+    - build-conda-vdf-env = virtual_dataframe.bin.__init__:main
+
+requirements:
+  host:
+    - python ==3.9
+    - pip >=22.3
+    - setuptools_scm
+
```

Comment:
```suggestion
```

Reply:
I can remove this line. Else, the build process failed.
It's because the setuptools_scm need a specific version of pip and python.

Reply:
This was about the extra empty line, it's fine.